README.md

```diff
@@ -3,17 +3,16 @@ PermonSVM - the PERMON SVM classifier
 
 PERMON project homepage: <http://permon.vsb.cz>  
 PermonSVM homepage: <http://permon.vsb.cz/permonsvm.htm>
-GitHub: <https://github.com/permon/permonsvm>
 
 Please use [GitHub](https://github.com/permon/permonsvm) for issues and pull requests.
 
-Features
---------
+Feature overview
+-----------------
 
 - The scalable parallel solution for the linear C-SVM 
```

Comment:
```suggestion
Scalable (parallel) solution for the linear C-SVM 
```

Reply:
I am not sure why you suggest parallel in brackets.